soap/src/java/com/zimbra/soap/mail/type/BulkAction.java

```diff
@@ -57,6 +58,13 @@ public static Operation fromString(String s) throws ServiceException {
     @XmlAttribute(name = MailConstants.A_FOLDER /* l */, required = false)
     private String folder;
 
+    /**
+     * @zm-api-field-tag read-flag
+     * @zm-api-field-description Read flag
```

Comment:
Perhaps:
Required if op="flag".  Set to "true" if all matching items should be marked as "read" otherwise set to false.